### PR TITLE
Add Achievement names!

### PR DESCRIPTION
**Pull request by BalloonChamp**
Almost all complete, still missing a few. Better than an hour ago :)

### CODE REVIEW

**Comment by mastercoms** [comment]
Will the rest be added in this pull request?

**Comment by BalloonChamp** [comment]
Umm I'll try

**Comment by BalloonChamp** [comment]
I think that there are duplicate achievements

File: src/main/java/net/glowstone/constants/GlowAchievement.java [hunk 1]
```diff
@@ -41,44 +41,44 @@ private static void set(Achievement achievement, String key, String fancyName) {
...
-        set(END_PORTAL, "theEnd");
-        set(THE_END, "theEnd2");
-        set(ENCHANTMENTS, "enchantments");
-        set(OVERKILL, "overkill");
-        set(BOOKCASE, "bookcase");
-        set(BREED_COW, "breedCow");
+        set(MINE_WOOD, "mineWood", "Getting Wood");
+        set(BUILD_WORKBENCH, "buildWorkBench", "Benchmarking");
+        set(BUILD_PICKAXE, "buildPickaxe", "Time to Mine!");
+        set(BUILD_FURNACE, "buildFurnace", "Hot Topic!");
```
Comment (mastercoms) [inline src/main/java/net/glowstone/constants/GlowAchievement.java]: No exclamation mark

File: src/main/java/net/glowstone/constants/GlowAchievement.java [hunk 2]
```diff
@@ -41,44 +41,44 @@ private static void set(Achievement achievement, String key, String fancyName) {
...
+        set(COOK_FISH, "cookFish", "Delicious Fish");
+        set(ON_A_RAIL, "onARail", "On a Rail");
+        set(BUILD_SWORD, "buildSword", "Time to Strike!");
+        set(KILL_ENEMY, "killEnemy", "Monster Hunter");
+        set(KILL_COW, "killCow", "Cow Tipper");
+        set(FLY_PIG, "flyPig", "When Pigs Fly");
+        set(SNIPE_SKELETON, "snipeSkeleton", "Sniper Duel");
+        set(GET_DIAMONDS, "diamonds", "DIAMONDS!");
+        set(DIAMONDS_TO_YOU, "diamondsToYou", "Diamonds to you!");
+        set(NETHER_PORTAL, "portal", "");
```
Comment (mastercoms) [inline src/main/java/net/glowstone/constants/GlowAchievement.java]: We Need to Go Deeper

File: src/main/java/net/glowstone/constants/GlowAchievement.java [hunk 3]
```diff
@@ -41,44 +41,44 @@ private static void set(Achievement achievement, String key, String fancyName) {
...
+        set(GHAST_RETURN, "ghast", "Return to Sender");
+        set(GET_BLAZE_ROD, "blazeRod", "Into Fire");
+        set(BREW_POTION, "potion", "Local Brewery");
+        set(END_PORTAL, "theEnd", "The End?");
+        set(THE_END, "theEnd2", "The End.");
+        set(ENCHANTMENTS, "enchantments", "Enchanter");
+        set(OVERKILL, "overkill", "Overkill");
+        set(BOOKCASE, "bookcase", "Librarian");
+        set(BREED_COW, "breedCow", "Repopulation");
+        set(SPAWN_WITHER, "spawnWither", "");
```
Comment (mastercoms) [inline src/main/java/net/glowstone/constants/GlowAchievement.java]: The Beginning?

File: src/main/java/net/glowstone/constants/GlowAchievement.java [hunk 4]
```diff
@@ -41,44 +41,44 @@ private static void set(Achievement achievement, String key, String fancyName) {
...
+        set(KILL_WITHER, "killWither", "The Beginning.");
+        set(FULL_BEACON, "fullBeacon", "Beaconator");
+        set(EXPLORE_ALL_BIOMES, "exploreAllBiomes", "Adventuring Time");
         set(SPAWN_WITHER, "spawnWither");
```
Comment (mastercoms) [inline src/main/java/net/glowstone/constants/GlowAchievement.java]: Duplicate line. Please remove.

File: src/main/java/net/glowstone/constants/GlowAchievement.java [hunk 5]
```diff
@@ -41,44 +41,44 @@ private static void set(Achievement achievement, String key, String fancyName) {
...
+        set(BUILD_BETTER_PICKAXE, "buildBetterPickaxe", "Getting an Upgrade");
+        set(COOK_FISH, "cookFish", "Delicious Fish");
+        set(ON_A_RAIL, "onARail", "On a Rail");
+        set(BUILD_SWORD, "buildSword", "Time to Strike!");
+        set(KILL_ENEMY, "killEnemy", "Monster Hunter");
+        set(KILL_COW, "killCow", "Cow Tipper");
+        set(FLY_PIG, "flyPig", "When Pigs Fly");
+        set(SNIPE_SKELETON, "snipeSkeleton", "Sniper Duel");
+        set(GET_DIAMONDS, "diamonds", "DIAMONDS!");
+        set(DIAMONDS_TO_YOU, "diamondsToYou", "Diamonds to you!");
```
Comment (mastercoms) [inline src/main/java/net/glowstone/constants/GlowAchievement.java]: Duplicate line. Please remove.

File: src/main/java/net/glowstone/constants/GlowAchievement.java [hunk 6]
```diff
@@ -41,44 +41,44 @@ private static void set(Achievement achievement, String key, String fancyName) {
...
+        set(NETHER_PORTAL, "portal", "");
+        set(GHAST_RETURN, "ghast", "Return to Sender");
+        set(GET_BLAZE_ROD, "blazeRod", "Into Fire");
+        set(BREW_POTION, "potion", "Local Brewery");
+        set(END_PORTAL, "theEnd", "The End?");
+        set(THE_END, "theEnd2", "The End.");
+        set(ENCHANTMENTS, "enchantments", "Enchanter");
+        set(OVERKILL, "overkill", "Overkill");
+        set(BOOKCASE, "bookcase", "Librarian");
+        set(BREED_COW, "breedCow", "Repopulation");
```
Comment (mastercoms) [inline src/main/java/net/glowstone/constants/GlowAchievement.java]: Duplicate line. Please remove.

File: src/main/java/net/glowstone/constants/GlowAchievement.java [hunk 7]
```diff
@@ -41,44 +41,44 @@ private static void set(Achievement achievement, String key, String fancyName) {
...
+        set(SPAWN_WITHER, "spawnWither", "");
+        set(KILL_WITHER, "killWither", "The Beginning.");
```
Comment (mastercoms) [inline src/main/java/net/glowstone/constants/GlowAchievement.java]: Duplicate line. Please remove.

File: src/main/java/net/glowstone/constants/GlowAchievement.java [hunk 8]
```diff
@@ -41,44 +41,44 @@ private static void set(Achievement achievement, String key, String fancyName) {
...
+        set(FULL_BEACON, "fullBeacon", "Beaconator");
```
Comment (mastercoms) [inline src/main/java/net/glowstone/constants/GlowAchievement.java]: Duplicate line. Please remove.

File: src/main/java/net/glowstone/constants/GlowAchievement.java [hunk 9]
```diff
@@ -41,44 +41,44 @@ private static void set(Achievement achievement, String key, String fancyName) {
...
-        set(KILL_WITHER, "killWither");
-        set(FULL_BEACON, "fullBeacon");
-        set(BREED_COW, "breedCow");
-        set(DIAMONDS_TO_YOU, "diamondsToYou");
-        set(OVERPOWERED, "overpowered");
+        set(KILL_WITHER, "killWither", "The Beginning.");
+        set(FULL_BEACON, "fullBeacon", "Beaconator");
+        set(BREED_COW, "breedCow", "Repopulation");
+        set(DIAMONDS_TO_YOU, "diamondsToYou", "Diamonds to you!");
+        set(OVERPOWERED, "overpowered", "OVERPOWERED!");
```
Comment (mastercoms) [inline src/main/java/net/glowstone/constants/GlowAchievement.java]: Overpowered

**Comment by mastercoms** [comment]
I've labeled all the lines that need fixing with the proper fix. Once you've fixed them, this will be merged. Thank you for your contribution.

**Comment by mastercoms** [comment]
I've done the fixes myself, and merged this! Thanks!

**Pull request merged**